lib/commands/test.js

```diff
@@ -20,7 +20,11 @@ module.exports = TestCommand.extend({
 
   rmTmp() {
     if (process.platform === 'win32') {
-      console.log(require('child_process').execSync('handle').toString());
+      // There is some kind of race condition where we intermittently get EBUSY
+      // deleting the temp directory on windows, which causes the test to fail.
+      // Unfortunately adding logging to try to track it down makes it go away,
+      // so let's just delay for a bit as a dirty workaround.
+      require('child_process').execSync('ping 127.0.0.1 -n 2 > nul');
```

Comment:
Uh, does this really work consistently? 😵 If so, we may have to borrow this for Forge tests.